src/backend/src/services/teams.services.ts

```
@@ -372,4 +372,38 @@ export default class TeamsService {
     const teamTypes = await prisma.teamType.findMany();
     return teamTypes;
   }
+
+  static async setTeamType(submitter: User, teamId: string, teamTypeId: string): Promise<Team> {
+    if (!isAdmin(submitter.role)) {
+      throw new AccessDeniedAdminOnlyException('set a team type');
+    }
+
+    const teamType = await prisma.teamType.findFirst({
+      where: { teamTypeId }
+    });
+
+    const team = await prisma.team.findUnique({
+      where: { teamId },
+      ...teamQueryArgs
+    });
+
+    if (!team) {
+      throw new NotFoundException('Team', teamId);
+    }
+
+    if (!teamType) {
+      throw new HttpException(400, 'Team type with the given ID does not exist');
```

can you throw a not found exception here as well